README.md

```diff
@@ -118,3 +118,19 @@ For development purposes:
 ```sh
 make serve
 ```
+
+### Linting JavaScript files
+
+To lint the JavaScript files use the following command:
+
+```sh
+npm run lint:js --fix
```

Comment:
Are you sure this works? I believe it should be `npm run lint:js -- --fix` to pass the `fix` option to the linter 🙂

I also feel like it would be more appropriate to not document `fix` as the default behaviour. If we really believe the best is to always auto fix, then we should add the option in the script itself.

```suggestion
npm run lint:js
```